File: lib/layer2_core/light/e_surface_container_lowest_light.dart
import '../../layer1_raw/design_token.dart';
import '../../layer1_raw/light/system.dart';
import '../../layer1_raw/light/standard.dart';
import '../../layer1_raw/light/blue.dart';
import '../../layer1_raw/light/green.dart';
import '../../layer1_raw/light/red.dart';
import '../../layer1_raw/light/nothing.dart';

class ESurfaceContainerLowestLight {
  static DesignToken getToken(dynamic activeTheme) {
    switch (activeTheme.toString().split('.').last) {
      case 'system': return csyl13;
      case 'standard': return cstl13;
      case 'blue': return cbl13;
      case 'green': return cgl13;
      case 'red': return crl13;
      case 'nothing': return cnol13;
      default: return cstl13;
    }
  }
}

File: android/src/main/kotlin/com/example/design_engine/layer2_core/light/ESurfaceContainerLowestLight.kt
package com.example.design_engine.layer2_core.light
import com.example.design_engine.R
import com.example.design_engine.layer3_logic.AppTheme
object ESurfaceContainerLowestLight {
    fun getResId(activeTheme: AppTheme): Int {
        return when (activeTheme) {
            AppTheme.SYSTEM -> R.string.csyl13
            AppTheme.STANDARD -> R.color.cstl13
            AppTheme.BLUE -> R.color.cbl13
            AppTheme.GREEN -> R.color.cgl13
            AppTheme.RED -> R.color.crl13
            AppTheme.NOTHING -> R.color.cnol13
        }
    }
}
